tests/test_autoscaling_service_lib.py

```diff
@@ -0,0 +1,48 @@
+import mock
```

Comment:
can you put this in tests/autoscaling/test_autoscaling_service_lib.py instead of starting a new file?

Or move the tests from that file into here? would like to keep the 1:1 mapping of python code and test code files to make it easier to tdd and discover

Reply:
oh, derp, I didn't see that file.